include/yaml_cpp_catkin/yaml_cpp_fwd.hpp

```diff
@@ -0,0 +1,13 @@
+/**
+ * @file yaml_cpp_fwd.hpp
+ * @author Maximilien Naveau (maximilien.naveau@gmail.com)
+ * @license License BSD-3-Clause
+ * @copyright Copyright (c) 2019, New York University and Max Planck Gesellschaft.
+ * @date 2019-09-30
+ */
+
+#pragma once
+
+#include <yaml-cpp/yaml.h>
+#include "yaml_cpp_catkin/yaml_eigen.h"
```

Comment:
Is there a reason for using `"..."` instead of `<...>` for these includes? In my understanding, `"..."` should only be used for relative includes.